WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,7 +1,5 @@
-Need a little writing inspiration? We have you covered with brand-new blogging prompts. Get a new prompt each day, write a post about it, and build a writing habit in no time.
+* [***] [Jetpack-only] The Insights view in Stats has been revamped with a cleaner look and a new set of cards to make it easier than ever to see at a glance how your site's performing. [#18945]
+* [*] Preview: Post preview now resizes to account for device orientation change. [#18921]
+* [***] [Jetpack-only] Enables QR Code Login scanning from the Me menu. [#18904]
+* [*] Reverted the app icon back to Cool Blue. Users can reselect last month's icon in Me > App Settings > App Icon if they'd like. [#18934]
```

Comment:
Note to self "Cool Blue" only applies to WordPress. Jetpack is, of course, back to it's brand-green. Remember to point this out when requesting editorialization (just in case)